.circleci/config.yml

```diff
@@ -71,176 +274,107 @@ jobs:
           - MYSQL_PASSWORD=ubuntu
     working_directory: ~/workbench
     environment:
-      # Drop to ~1.5G; the local API server starts two Java processes (the API
-      # server and the cron emulator). The default machine has 4GB, so we want
-      # the Java memory limit to be below half of that: other things on the
-      # machine need memory as well. Using 2G here manifested in 137 errors.
-      # If this limit proves to be too low, an alternative would be to use a
-      # larger resource class:
-      #   https://circleci.com/docs/2.0/configuration-reference/#resource_class
```

Comment:
with a `medium+` circleci container and `GRADLE_OPTS: "-Dorg.gradle.daemon=false"`, OOM issue should have been resolved. Thus, removed comments.

Reply:
We need a comment above explaining why it has been set to medium+